dev/how-to/get-started/local-cluster/install-from-docker-compose.md

```diff
@@ -11,6 +11,13 @@ This document describes how to quickly deploy a TiDB testing cluster with a sing
 
 With Docker Compose, you can use a YAML file to configure application services in multiple containers. Then, with a single command, you can create and start all the services from your configuration.
 
+## Alternatives
+
+If you want to use docker but just test MySQL compatibility or otherwise run just a single TiDB server (without TiKV), there is a [3rd-party image](https://gitlab.com/jniltinho/tidb-alpine) for that.
```

Comment:
```suggestion
If you want to use Docker but just test MySQL compatibility or otherwise run just a single TiDB server (without TiKV), there is a [3rd-party image](https://gitlab.com/jniltinho/tidb-alpine) for that.
```